resources/build/sentry-control.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#
+# This script centralises sentry symbol uploads and releases
+# Note: We must install sentry-cli on the build agents
+#
+
+
+## TODO: centralise debug information file (dif) uploads
+
+
+function isSentryConfigured() {
+  if [ -z ${SENTRY_AUTH_TOKEN-} ] || [ -z ${SENTRY_ORG} ] || [ -z ${SENTRY_URL} ]; then
```

Comment:
Is it 
`${SENTRY_AUTH_TOKEN-}` or 
`${SENTRY_AUTH_TOKEN}`?

Reply:
The `-` is right. The other two tests need it too! 😊

Reply:
https://stackoverflow.com/a/40630951/1836776